mavros/src/plugins/waypoint.cpp

```diff
@@ -232,7 +232,12 @@ class WaypointPlugin : public plugin::PluginBase {
 
 	/* -*- rx handlers -*- */
 
-	// WaypointItem is MISSION_ITEM child
+	/**
+	 * @brief handle MISSION_ITEM mavlink msg
+	 * handles and stores mission items when pulling waypoints
+	 * @param msg       Received Mavlink msg
```

Comment:
Please use tab instead of whitespaces. (use the uncrustify tool)

Reply:
I did use uncrustify but i guess it doesn't fix tabs that aren't at the start of a line.
I've fixed this